locales/gaia-toolbar.en-US.properties

```diff
@@ -0,0 +1 @@
+gaiaToolbarMore.ariaLabel = More
```

Comment:
After having a conversation with Stas, this should serve as an example for users trying to use localization for this component. The messages can either be defined in the page messages file or additional .properties file similar to this: https://github.com/mozilla-b2g/gaia/blob/master/apps/system/index.html#L124-L125